ui/src/org/pentaho/di/ui/spoon/Spoon.java

```diff
@@ -4932,6 +4932,11 @@ public boolean quitFile( boolean canCancel ) throws KettleException {
     }
 
     if ( exit ) {
+      // on windows [...].swt.ole.win32.OleClientSite.OnInPlaceDeactivate can
+      // cause the focus to move to an already disposed tab, resulting in a NPE
+      // so we first move the focus to somewhere else
+      this.selectionLabel.forceFocus();
```

Comment:
It's the `close()` that will do the actual quitting of the application, so there is no reason for the control be already disposed. Also, that label exists from the beginning to the end of the application, so it shouldn't be null either. But I can add the check, to be extra safe.